.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,14 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
```

Comment:
Please remove this file

Reply:
Dunno if anyone _will_ use it but they **can** use it. If you autogenerate, VSCode will create a according `Dockerfile`, at least it did for me, which is not necessary in our use case.